vernac/vernacinterp.ml

```diff
@@ -184,34 +201,27 @@ and vernac_load ~verbosely fname =
     let proof_mode = Option.map (fun _ -> get_default_proof_mode ()) stack in
     match parse_sentence proof_mode input with
     | None -> stack, pm
-    | Some stm ->
-      let stack, pm = v_mod (interp_control ~st:{ st with Vernacstate.lemmas = stack; program = pm }) stm in
+    | Some { loc; v = cmd } ->
+      let interp_fn ?loc ~st ~atts expr =
+        let e = Synterp.synterp ?loc ~atts expr in
+        interp_expr ?loc ~atts ~st e
+      in
+      (*
+      let expr = Synterp.synterp ?loc ~atts:stm.attrs stm.expr in
+      let stm = CAst.make ?loc { attrs = stm.attrs; control = stm.control; expr } in
+      *)
```

Comment:
What does this comment mean?

Reply:
It means you're looking at a draft PR ;)

Reply:
Fixed.